examples/cartrade/README.md

```diff
@@ -29,6 +29,7 @@ Before you begin, you need to check that you have all the prerequisites installe
 	```
 	./script-start-ledgers.sh
 	```
+	- (NOTICE: Before executing the above, your account need to be added to the docker group (`usermod -a -G docker YourAccount` from root user))
```

Comment:
typo: your account need -> your account needs
Would you correct about this point?